classes/user/UserSettingsDAO.inc.php

```diff
@@ -18,9 +18,6 @@
 namespace PKP\user;
 
 use APP\core\Application;
-use PKP\db\DAORegistry;
-
-use PKP\db\DAOResultFactory;
 
 class UserSettingsDAO extends \PKP\db\DAO
```

Comment:
Maybe not in this PR, but it would be nice to do away with this class, since it breaks convention with other DAOs. I think some of the methods are covered in the regular DAO and some could be moved there.

Reply:
Yes, I thought the same but needed to control scope on this one. I think we'll have to do some future work on things like relationship tables in particular.